src/Npgsql/PreparedTextReader.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Npgsql.Internal;
+
+namespace Npgsql
```

Comment:
:fire: :fire: :fire: :fire: :fire: 
:fire: :fire: :fearful: :fire: :fire: 
:fire: :fire: :fire: :fire: :fire: 

(file-scoped namespace :))

Reply:
That's a 2 month-old pr, it's fine)